src/store/getters/state-getters.ts

```diff
@@ -517,62 +517,84 @@ export const hasContactInfoChanged = (state: StateIF): boolean => {
   )
 }
 
-/** The dynamic Office Type based on the current filing. */
-export const officeType = (state: StateIF): OfficeTypes => {
-  if (isChangeFiling(state) || isConversionFiling(state)) return OfficeTypes.BUSINESS_OFFICE
-  if (isAlterationFiling(state) || isCorrectionFiling(state)) return OfficeTypes.REGISTERED_OFFICE
-}
-
 /** True if any office address has changed. Applies to corrections, change and conversion filings only. */
 export const hasOfficeAddressesChanged = (state: StateIF): boolean => {
   const isExpectedFiling = isCorrectionFiling(state) || isChangeFiling(state) || isConversionFiling(state)
-  const isAddressChanged = (mailingChanged(state) || deliveryChanged(state) ||
+  const hasMailingDeliveryChanged = hasMailingChanged(state) || hasDeliveryChanged(state)
+  const isChangeOrConversionFiling = isChangeFiling(state) || isConversionFiling(state)
+  const hasRecMailingDeliveryChanged = hasRecMailingChanged(state) || hasRecDeliveryChanged(state)
+  const hasAddressChanged = (
+    hasMailingDeliveryChanged ||
     // exclude Records Address conditions from Change or Conversion filing
-    (!isChangeFiling(state) && !isConversionFiling(state) && (recMailingChanged(state) || recDeliveryChanged(state)))
+    (!isChangeOrConversionFiling && hasRecMailingDeliveryChanged)
   )
 
-  return (isExpectedFiling && isAddressChanged)
+  return (isExpectedFiling && hasAddressChanged)
```

Comment:
Refactored for simpler logic. (I needed to update this anyway since the original line was too long.)